### PR TITLE
Try to fix tiles editor on pre-M (again...)

### PR DESCRIPTION
**Pull request by Kamin4ri**
Signed-off-by: Kamin4ri <kamin4ridev@gmail.com>

### CODE REVIEW

File: app/src/main/java/tk/wasdennnoch/androidn_ify/notifications/qs/QSTileHostHooks.java [hunk 1]
```diff
@@ -228,7 +287,11 @@ public static String getDefaultTilesPref() {
             specs.add("inversion");
             specs.add("cell");
             specs.add("airplane");
-            specs.add("dnd");
+
+            // DND tile was added only on M!
+            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M)
```
Comment (wasdennnoch) [inline app/src/main/java/tk/wasdennnoch/androidn_ify/notifications/qs/QSTileHostHooks.java]: It was added in M and you add it only below M :p

**Comment by wasdennnoch** [comment]
I've seen that on LP the tile host first gets a spec string from the secure settings and uses a default value is the stored list is empty (which it always is), maybe this can help too. I also left a line note because of a small mistake ;)

**Comment by wasdennnoch** [comment]
nvm you fixed it

**Comment by Kamin4ri** [comment]
Now we need to find a way to preserve these changes (a reboot does away with all the tiles). And also, proper interaction with GravityBox - we can't add gb's tiles yet

**Comment by wasdennnoch** [comment]
I think the only way it to hook into GB itself, but that will be tricky.

**Comment by Kamin4ri** [comment]
A xda user sent me his Xposed log... it looks like the tiles spec was being brute-forced. I'm taking a look at the code right now (QSTileHostHooks)

**Comment by wasdennnoch** [comment]
sort of brute force. I'm just testing various tile specs and save the working ones.

**Comment by Kamin4ri** [comment]
Update. Rebooting does not clear the tiles. However the battery tile sometimes appears only when the phone's charging (it sticks after the phone has been unplugged, though).

**Comment by wasdennnoch** [comment]
I noticed the disappearing battety too when opening the QS panel and locking the screen. The battery is in the QuickQS header but not in the actual panel.

**Comment by Kamin4ri** [comment]
Well, my knowledge about Android coding is very basic, so I dont really know what to do next :p

**Pull request merged**